CoreScriptsRoot/Modules/BackpackScript.lua

```diff
@@ -1656,6 +1767,17 @@ local function OnVREnabled(prop)
 			end
 		end)
 
+		-- Turn off dragging when in VR
+		for _, slot in pairs(Slots) do
+			slot:SetClickability(false)
+		end
+
+		-- Reset the constants!
```

Comment:
What happens if VR is disabled? Is that even possible? [To get here UserInputService.VREanbled needs to be true.](https://github.com/ROBLOX/Core-Scripts/pull/471/files#diff-73dcf66ad2dbe31e15c6c4c51de4d67bL1578), so IsVR will always be true at this point, and the constants will not change.